backend/app/routes/faq/faq.js

```diff
@@ -0,0 +1,39 @@
+const {authMiddleware, generateJWT} = require('../../../helpers/middlewares/auth');
+const express = require ("express");
+const FAQ = require('../../models/faq');
+const router = express.Router();
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+    try {        
+        const {
+            question,
+            answer,
+            isActive,
+            tags,
+            }=req.body;
+
+        const faq = {};
+
+        faq.question = question;
+        faq.answer = answer;
+        if(isActive) faq.isActive = isActive;
+        if(tags) faq.tags = tags; 
```

Comment:
These can be done in a single assignment, no need of first consuming them in variables and then in json